code/socialDistribution/views.py

```diff
@@ -521,6 +541,24 @@ def share_post(request, id):
 
     return redirect('socialDistribution:home')
 
+def copy_link(request, id):
+    """
+        Allows user to copy a post's link
+    """
+
+    # TODO:
+    #     * make copy notification prettier 
+    #     * add copy link for remote posts 
+
+    author = LocalAuthor.objects.get(user=request.user)
+    post = LocalPost.objects.get(id=id)
+    link = post.get_local_shareable_link()
+    pyperclip.copy(link)
```

Comment:
This line is crashing my site 

![image](https://user-images.githubusercontent.com/47841864/143666695-4e083396-4f3c-4386-877b-bd8bffcb29fe.png)

I'm running Ubuntu 21.04. Maybe we should just keep it simple and provide the text and the user can copy it themselves?